File: cipher.c
#include <stdio.h>
#include <stdlib.h>

void main(){
	printf("Content-Type: text/plain\r\n\r\n");
	if (getenv("HTTPS")){
		printf("%s (%s)", getenv("SSL_PROTOCOL"), getenv("SSL_CIPHER"));
	}
}

File: cipher.py
#!/usr/bin/env python3
import os

print("Content-Type: text/plain\r\n\r")
if os.environ.get("HTTPS"):
	print(f'{os.environ.get("SSL_PROTOCOL")} ({os.environ.get("SSL_CIPHER")})' )
else:
	print("-")

# SSL_PROTOCOL